classes/fsl-eula-unpack.bbclass

```diff
@@ -157,11 +157,21 @@ python fsl_bin_do_unpack() {
     bb.note("Checking LIC_FILES_CHKSUM for Freescale EULA consistency...")
     if found > 1:
         bb.warn("The package contains multiple Freescale EULA-licensed archives. The consistency logic may not be able to detect a EULA problem.")
+    (layer_license, licenses, md5sums, found_layer_licenses, found_package_licenses) = findNxpEulaLicenses(d)
+    if not found_layer_licenses:
+        bb.fatal("The Freescale layer EULA '%s' is not listed in LIC_FILES_CHKSUM '%s'."
+                 % (layer_license, licenses))
+    if not found_package_licenses:
+        bb.fatal("A valid package EULA with md5sum in %s was not found in LIC_FILES_CHKSUM '%s'."
+                 % (md5sums.split(), licenses))
+}
+
+def findNxpEulaLicenses(d):
```

Comment:
Python uses `find_nxp_eula_licenses(d):` so the snake_case.

Reply:
Haha, old habit.